extensions/wikia/StoriesLinkTag/StoriesLinkTagHelper.class.php

```diff
@@ -121,12 +121,21 @@ protected function getArticleDescription( $data ) {
 	 * @return string
 	 */
 	protected function getArticleImageUrl( $data ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)